lib/Notes/NotePopupModal/NotePopupModal.js

```diff
@@ -111,13 +113,22 @@ const NotePopupModal = ({
     setPopupNote(note);
     mutator.popupNote.update({ id: note.id });
     setIsOpen(shouldShowPopup);
-  }, [mutator.popupNote, getPopupNoteFromNotes, getSessionStorageNoteInfo]);
+    sessionStorage.removeItem(getSessionStorageKey(note.id));
+  }, [
+    mutator.popupNote,
+    getPopupNoteFromNotes,
+    getSessionStorageNoteInfo,
+    getSessionStorageKey,
+  ]);
 
   useEffect(() => {
-    if (!popupNote && entityId) {
-      mutator.NotePopupModal_assignedNotes.GET().then(handleGetNotesRequest);
+    if (!popupNote && entityId && !isNotesLoading.current) {
+      mutator.NotePopupModal_assignedNotes.GET()
+        .then(handleGetNotesRequest);
+
+      isNotesLoading.current = true;
     }
-  }, [entityId, handleGetNotesRequest, popupNote]);
+  }, [entityId, handleGetNotesRequest, popupNote, mutator.NotePopupModal_assignedNotes]);
```

Comment:
Should it be `mutator.NotePopupModal_assignedNotes.GET`?

Reply:
For some reason eslint still requires `mutator.NotePopupModal_assignedNotes` to be passed even if `mutator.NotePopupModal_assignedNotes.GET` is in dependencies